src/senaite/core/exportimport/genericsetup/adapters.py

```diff
@@ -219,12 +220,21 @@ def set_node_value(self, node):
         self.set_field_value(data, filename=filename)
 
     def get_archive_path(self):
-        """Get the unified archive path
+        """Calculate the archive path from the context
         """
         site = self.environ.getSite()
         site_path = api.get_path(site)
         obj_path = api.get_path(self.context)
-        return obj_path.replace(site_path, SITE_ID, 1)
+        # remove the portal path
+        rel_path = obj_path.replace(site_path, "")
```

Comment:
Note the following:

```python
(Pdb++) site_path
'/senaite'
(Pdb++) obj_path
'/senaite/senaite_storage/SC0120335'
(Pdb++) obj_path.replace(site_path, "")
'_storage/SC0120335'
```

Reply:
Yes, the path inside the archive starts always with `senaite`, whereas the object path of the imported site might be `/senaite` or `/senaite2` or `/my-senaite-site` etc.
Therefore, we need to translate the path to match the file path inside the archive, e.g. `senate/clients/client-1/report.pdf`

Reply:
Yes, I understand. What I wanted to say is that if the object path starts with "/senaite/senaite_" (e.g. an object from filestorage), this replacement won't work as expected, cause the function will not remove the portal path only, but also part of the relative path of the context.

Reply:
ah, now I got it! Man, you have eyes like an eagle!:)